### DISCUSSION TITLE
Real-time multiplayer scores don't properly account for NF

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16557] [Q&A] [answered]
Using Standardised scoring, the scores from my player results screen give a total of 5,132,728. However, the score bar at the top seems to indicate a total of 6,438,041. Viewing the spectator screen, this appears to be resulting from No Fail not properly using the maximum of 500k. It doesn't matter whether NF is debuffed or not, the scores should corroborate. (Source: https://youtu.be/DDVeIYY3pYw?t=1675)
Version 2022.108.0
~~Last mp report from me today~~
![image](https://user-images.githubusercontent.com/74262407/150626254-032d8db9-f433-4408-8c3d-621d5cbad5a5.png)

**Comment by bdach** [comment, marked as answer]
#16562
